site/public/js/configuration.js

```diff
@@ -3,13 +3,24 @@ $(document).ready(function() {
     updateEmailSeatingOption();
 
     function updateForumMessage() {
+        updateForumCategoryWarning();
         if ($("#forum-enabled").is(":checked")) {
             $("#forum-enabled-message").show();
         } else {
             $("#forum-enabled-message").hide();
         }
     }
 
+    function updateForumCategoryWarning() {
+        if ($("#forum-category-warning").length) {
+            if ($("#forum-enabled").is(":checked")) {
```

Comment:
Replace the if else logic with $("#forum-enabled").toggle(); instead of using show and hide

Reply:
Is there a reason to use `toggle()` over `show()` and `hide()`? Right now we are using `updateForumMessage()` to hide/show the message during document load. I think even if we were to switch to toggle, we would still need to make changes in `Configuration.twig`

Reply:
using toggle() is basically the same as having the if else statement with the show and hide but jquery does the work for us. I recommend using it and setting the elements to hidden by default.

You could either do this during the document onload on client side or in the twig create
the elements as hidden.

Either case I think its cleaner to use toggle and do a second client side check to see if we should render the message onload or not

Reply:
Changed to use toggle. I choose to hide elements in twig.